packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -5868,4 +5882,53 @@ export default class Meeting extends StatelessWebexPlugin {
       failure: `${LOG_HEADER} disable bnr failure, `
     });
   }
+
+  /**
+   * starts keepAlives being sent
+   * @returns {void}
+   * @private
+   * @memberof Meeting
+   */
+  startKeepAlive = () => {
+    if (this.keepAliveTimerId) {
+      LoggerProxy.logger.error('Meeting:index#startKeepAlive --> keepAliveTimerId already exists');
+
+      return;
+    }
+    if (!this.joinedWith?.keepAliveUrl) {
+      LoggerProxy.logger.error('Meeting:index#startKeepAlive --> no keepAliveUrl');
+
+      return;
+    }
+    if (!this.joinedWith?.keepAliveSecs) {
+      LoggerProxy.logger.error('Meeting:index#startKeepAlive --> no keepAliveSecs');
+
+      return;
+    }
+    if (this.joinedWith.keepAliveSecs <= 1) {
+      LoggerProxy.logger.error('Meeting:index#startKeepAlive --> keepAliveSecs <= 1');
+
+      return;
+    }
+    const {keepAliveUrl} = this.joinedWith;
+    const keepAliveInterval = (this.joinedWith.keepAliveSecs - 1) * 750;
+
+    this.keepAliveTimerId = safeSetInterval(() => { this.meetingRequest.keepAlive({keepAliveUrl}); }, keepAliveInterval);
```

Comment:
not sure if we need to use the "safe" variant here. Plugin-meetings seems to be full of calls to the standard setTimeout/setInterval and not a single call to the safe ones. I think it's because plugin-meetings doesn't support being run in node.js

Reply:
ok, will use the normal one

Reply:
done